biome.json

```diff
@@ -35,7 +35,6 @@
         "noParameterAssign": "off",
         "noUselessElse": "off",
         "noVar": "off",
-        "useExponentiationOperator": "off",
```

Comment:
Can we leave this syntax for later. I was introduced with es6 or es7

Reply:
Done